sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -277,14 +268,52 @@ case class Divide(left: Expression, right: Expression)
           if (${eval1.isNull}) {
             ${ev.isNull} = true;
           } else {
-            ${ev.value} = $divide;
+            ${ev.value} = $division;
           }
         }""")
     }
   }
 }
 
 @ExpressionDescription(
+  usage = "a _FUNC_ b - Divides a by b.",
```

Comment:
we should mention this is a fraction division, i.e. the parameter must be fraction type and the result is also fraction.